app/services/queries/reviews_turnaround_per_project_query.rb

```diff
@@ -0,0 +1,60 @@
+module Queries
+  class ReviewsTurnaroundPerProjectQuery
+    attr_reader :time_interval
+
+    def initialize(time_interval:)
+      @time_interval = time_interval
+    end
+
+    def last_review_created_at
+      Events::Review.order(:created_at).pluck(:created_at).last
```

Comment:
I think it's better to do `.first.created_at` instead of `pluck` because at some point there will be many reviews, and will take more memory space than just one object.

Reply:
Fixed it. Thanks!